MiniProjectUIKit/ViewController/DetailMenuViewController.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  DetailMenuViewController.swift
+//  MiniProjectUIKit
+//
+//  Created by Vincent Saranang on 06/12/24.
+//
+
+import UIKit
+
+class DetailMenuViewController: UIViewController {
+    private let meal: Meal
+
+    init(meal: Meal) {
+        self.meal = meal
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
```suggestion
        super.init(coder: coder)
```
This might lead to a crash if triggered. Consider to change this code.